bedrock/mozorg/templates/mozorg/contact/spaces/includes/nav.html

```diff
@@ -29,11 +29,6 @@ <h4 class="mzp-c-sidemenu-title {% if current == 'berlin' %}mzp-is-current{% end
           <a href="{{ url('mozorg.contact.spaces.berlin') }}">Berlin</a>
         </h4>
       </li>
-      <li>
```

Comment:
There's a functional test here that will need updating to remove `paris` from the list of slugs:

https://github.com/mozilla/bedrock/blob/main/tests/functional/test_contact.py#L22